packages/xstate-react/src/useInterpret.ts

```diff
@@ -85,7 +90,8 @@ export function useInterpret<
       actions,
       activities,
       services,
-      delays
+      delays,
+      input
     };
     const machineWithConfig = machine.withConfig(machineConfig, () => ({
```

Comment:
The initial render should already contain the "current" input value - we shouldn't have to wait for the first commit phase to get access to it.

Basically this should hold true:
```js
const [state] = useMachine(m, { input: { foo: 'bar' } })
expect(state.input.foo).toBe('bar')
```